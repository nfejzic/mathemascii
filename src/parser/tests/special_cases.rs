use super::Snapshot;

super::test_snap!(matrix, "[[a,b],[c,d]]");
super::test_snap!(column_vector, "((a),(b))");
super::test_snap!(augmented_matrix, "[[a,b,|,c],[d,e,|,f]]");
super::test_snap!(matrix_layout, "{(2x,+,17y,=,23),(x,-,y,=,5):}");

super::test_snap!(derivatives, "f'(x) = dy/dx");
