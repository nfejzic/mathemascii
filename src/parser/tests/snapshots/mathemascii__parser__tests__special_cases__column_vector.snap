---
source: src/parser/tests/special_cases.rs
expression: "Snapshot((input, math))"
---
((a),(b))

Expression {
| OpenParen
| | Expression {
| | | OpenParen
| | | | Expression {
| | | | | Variable("a")
| | | | }
| | | |
| | | CloseParen
| | }
| | 
| | Expression {
| | | Other(Comma)
| | }
| | 
| | Expression {
| | | OpenParen
| | | | Expression {
| | | | | Variable("b")
| | | | }
| | | |
| | | CloseParen
| | }
| |
| CloseParen
}


